### PR TITLE
Fix the syntax of language ids.

### PR DESCRIPTION
**Pull request by JimiC**
This is a trivial PR but I can't believe that this slipped me by.

### CODE REVIEW

**Comment by JimiC** [comment]
Bite me `Codecov`.

File: src/icon-manifest/languages.ts [hunk 1]
```diff
@@ -58,112 +58,112 @@ export const languages: ILanguageCollection = {
...
   json5: { ids: 'json5', defaultExtension: 'json5' },
   julia: { ids: ['julia', 'juliamarkdown'], defaultExtension: 'jl' },
-  kos: { ids: ['kos'], defaultExtension: 'ks' },
-  kotlin: { ids: ['kotlin'], defaultExtension: 'kt' },
+  kos: { ids: 'kos', defaultExtension: 'ks' },
+  kotlin: { ids: 'kotlin', defaultExtension: 'kt' },
   less: { ids: 'less', defaultExtension: 'less' },
-  lisp: { ids: ['lisp'], defaultExtension: 'lisp' },
+  lisp: { ids: 'lisp', defaultExtension: 'lisp' },
   literatehaskell: { ids: ['literate haskell'], defaultExtension: 'lhs' },
```
Comment (robertohuertasm) [inline src/icon-manifest/languages.ts]: I guess this one can also be simplified.

Comment (JimiC) [reply src/icon-manifest/languages.ts]: Yeah. Just leave it for today. I need to fix the code coverage aswell.

**Comment by robertohuertasm** [comment]
@JimiC, it seems that there are conflicts with this branch. Could you fix them so I can merge it? :+1:

**Comment by robertohuertasm** [comment]
I've already rebased it. No problem.

**Pull request merged**